index.html

```diff
@@ -4,7 +4,70 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap"
+      rel="stylesheet"
+    />
     <title>setup-mb-skeleton</title>
+    <link rel="stylesheet" href="style.css" />
   </head>
-  <body></body>
+  <body>
+    <header>
+      <nav>
+        <a href="#" class="logo">My logo</a>
+        <a href="#"><img src="images/Menu.svg" alt="hamburger" /></a>
+      </nav>
+    </header>
```

Comment:
- [ ] Good job working on the requested changes, however, your design has not matched the Figma template. 
Your design
![Capture](https://user-images.githubusercontent.com/85095702/158906383-30f156c3-c432-4af3-b1a8-474fa4eecfed.PNG)

Figma template

![figma](https://user-images.githubusercontent.com/85095702/158906426-5b9c14f2-ddfc-4f91-b385-2d5dae5faad6.PNG)

Kindly make your design responsive and remove the horizontal scroll

- [ ] Also kindly ensure that the font (Size, color and weight) match that of the Figma template

Reply:
Okay! great

Reply:
the scroll is now fixed! For my first attempt, I was using all those little images provided in Figma and my design was looking quite the same as the one provided but my first reviewer told me I should use the background provided. Since I have been using it maybe because of it size It doesn't fit as much as the sample look. and when I try to adjust the background size and/or position the responsiveness of it is not okay at all...  @phelian23